views/layouts/default.lavender

```diff
@@ -83,4 +83,7 @@ body
                 a(href='/guide.html#langref') Language Reference
           li(class='#{active_class_about}')
             a(href='/about.html') About
+          li(class='#{active_class_download}')
```

Comment:
you don't need to interpolate the active_class_download variable as a string. Within the parenthesis all variables are parsed normally. You can just write li(class=active_class_download)